SimpleITK/utilities/vtk.py

```diff
@@ -7,10 +25,6 @@
 def sitk2vtk(img, debugOn=False):
     """
     Function to convert a SimpleITK image to a VTK image.
-    Written by David T. Chen from the National Institute of Allergy
```

Comment:
There is much further to go on the procedure documentation. But that is another PR @dave3d